src/lib/swapper/swappers/OsmosisSwapper/utils/helpers.ts

```diff
@@ -216,14 +228,22 @@ export const performIbcTransfer = async (
 
   const bip44Params = adapter.getBIP44Params({ accountNumber })
 
+  return {
+    tx,
+    addressNList: toAddressNList(bip44Params),
+    chain_id: fromChainId(adapter.getChainId()).chainReference,
+    account_number: ibcAccountNumber.toString(),
```

Comment:
I see this snake case is originating in hdwallet-core, any idea why though?

Reply:
This is the Cosmos SDK message vernacular:

<img width="622" alt="image" src="https://github.com/shapeshift/web/assets/17035424/cc8eb5c8-e8de-49da-bfd5-39a646df5388">

Which we're pretty much passing through hdwallet as-is:

https://github.com/shapeshift/web/blob/fa2b0b5185ab467eb9820df920e1b48d387a02c5/packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts#L254

This makes sense at first, but while reading through the hdwallet implementation, found this:

<img width="790" alt="image" src="https://github.com/shapeshift/web/assets/17035424/22a06df3-895d-4bbf-8dae-1dabf31847a3">

We should be safe to make this `accountNumber` for consistency, though this will require an hdwallet bump so I would probably avoid it to avoid breaking things

Reply:
Piggybacking into this, this is actually correct: https://github.com/shapeshift/lib/pull/1159